webpack/assets/javascripts/react_app/components/BreadcrumbBar/components/Breadcrumb.js

```diff
@@ -21,6 +29,9 @@ const Breadcrumb = ({
         const { caption, caption: { icon, text } } = item;
         const overrideTitle = active && titleReplacement;
         const itemTitle = overrideTitle || text || caption;
+        const inner = (active
+                       ? (<span title={itemTitle} data-placement="bottom">{truncate(itemTitle, maxLength)}</span>)
```

Comment:
please use patternfly-react's [tooltip](http://patternfly-react.surge.sh/patternfly-3/index.html?knob-Tooltip=%3Cstrong%3EHoly%20guacamole%21%3C%2Fstrong%3E%20Check%20this%20info.&knob-Placement=right&knob-Trigger=hover%20focus&selectedKind=patternfly-react%2FWidgets%2FTooltip&selectedStory=Tooltip&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybooks%2Fstorybook-addon-knobs), the title prop doesn't work well with react (it's a jquery implementation)

Reply:
Done